recipes/psiclass/meta.yaml

```
@@ -8,17 +8,20 @@ source:
   sha256: c60c0f33f9a076b97ccdba68b9140a5fb6936a5c5e309d70b3d0cf558828f73d
 
 build:
-  number: 2
+  number: 3
+  run_exports:
+    - {{ pin_subpackage("myrecipe", max_pin="x.x") }}
 
 requirements:
   build:
     - make
     - {{ compiler('cxx') }}
     - {{ compiler('c') }}
   host:
-    - samtools 0.1.19.*
+    - zlib >= 1.2.11
```

samtools itself is still needed, isn't it?

```suggestion
    - zlib >= 1.2.11
    - samtools 0.1.19.*
```